net/net-reactor.cpp

```diff
@@ -323,7 +326,7 @@ int net_reactor_insert(net_reactor_ctx_t *ctx, int fd, int flags) {
     return 0;
   }
 
-  flags &= EVT_NEW | EVT_NOHUP | EVT_LEVEL | EVT_RWX;
+  flags &= EVT_NEW | EVT_NOHUP | EVT_LEVEL | EVT_RWX | EPOLLONESHOT;
```

Comment:
Maybe add own flag for EPOLLONESHOT as it done for e.g. EVT_READ <-> EPOLLIN?

Reply:
I think it's strange that we have own flags which map to library flags (EVT_READ <-> EPOLLIN and so on). So, I think it's better not to create new own flags. For example, EPOLLEXCLUSIVE doesn't have other own flag too

Reply:
Sure